.travis.yml

```diff
@@ -2,6 +2,10 @@ language: ruby
 
 sudo: false
 
+# "gemfile" is required for "allow_failures" option,
+# see https://docs.travis-ci.com/user/customizing-the-build/#matching-jobs-with-allow_failures
+gemfile:
```

Comment:
not sure if this is needed, as our `.travis.yml` already do not follow 
> When matching jobs against the definitions given in allow_failures, all conditions in allow_failures must be met exactly, and all the keys in allow_failures element must exist in the top level of the build matrix (i.e., not in matrix.include).

or I'm missing something :)

Reply:
🤷‍♂️

Reply:
from the [docs](https://docs.travis-ci.com/user/customizing-the-build/#matching-jobs-with-allow_failures)
> **and all the keys in allow_failures element must exist in the top level of the build matrix (i.e., not in matrix.include).**

according to this information we should have `rvm` key on the top level of build matrix(?) but not in matrix.include

I used this example as basis(see the link to docs):
```yaml
language: php
php:
- 5.6
- 7.0
env: # important!
matrix:
  include:
  - php: 7.0
    env: KEY=VALUE
  allow_failures:
  - php: 7.0
    env: KEY=VALUE
```
>Without the top-level env, no job will be allowed to fail

Reply:
> 🤷‍♂️

@dblock sorry, I'm not certainly sure that you mean. It's something like "I'm not sure what are you talking about" or "I don't know also". If this is the latter then ignore my comment above :)

Reply:
I meant I don't know also, trust whatever you do here